tests/Integration/features/delete-notifications.feature

```diff
@@ -9,6 +9,18 @@ Feature: delete-notifications
     Given user "test1" has notifications
     Then user "test1" has 3 notifications
     And delete first notification
+    And status code is 200
+    And user "test1" has 2 notifications missing the first one
+
+  Scenario: Delete same notification twice
+    Given user "test1" has notifications
+    Given user "test1" has notifications
+    Given user "test1" has notifications
+    When user "test1" has 3 notifications
+    And delete first notification
+    And status code is 200
+    And delete same notification
+    And status code is 404
```

Comment:
This tests fails. But that is to be expected if you look at https://github.com/nextcloud/notifications/pull/21/files#diff-30970b8579f39524067cc33fa0f56833L157